.github/workflows/publish-hotfix-release.yml

```diff
@@ -0,0 +1,267 @@
+#################################################################################
+#  Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+
+---
+name: "Publish hotfix release"
+
+# Temporary workflow to enable the release of bugfix versions.
+# Should be reworked as this file contains duplicate code from draft-new-release and publish-new-release
+# This workflow, as is, should be executed after the hotfix branch is created and all needed PRs are merged
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The hotfix version you want to release.'
+        required: true
+
+jobs:
+  # Gate: Check release version presence
+  validate-release:
+    name: Validate if hotfix can be released
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref_name, 'hotfix/')
```

Comment:
Using a dedicated GH Action for checking if a tag exists is needlessly complicated. how about `git tag -l | grep $TAGNAME` or `git rev-parse $TAGNAME`?

Reply:
I figured using a dedicated action for that would make the WF easier to read. Also wanted to avoid writing bash code because it gets tricky with GH sometimes. I refactored this step in [3766971](https://github.com/eclipse-tractusx/tractusx-edc/pull/1511/commits/3766971cc9ac180aa7bd43f8a0e8b7fef0ab21b3) since the action was not accepted by dash.